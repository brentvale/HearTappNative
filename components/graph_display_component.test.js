import 'react-native';
import React from 'react';
import GraphDisplayComponent from './graph_display_component';

import renderer from 'react-test-renderer';

test('renders properly', () => {
	const tree = renderer.create(
		<GraphDisplayComponent />
	).toJSON();
	expect(tree).toMatchSnapshot();
})