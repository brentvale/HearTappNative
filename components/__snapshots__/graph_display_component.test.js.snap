// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "textAlign": "center",
      }
    }
  >
     bpm
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#f2f2f2",
        "height": 150,
        "justifyContent": "center",
        "marginLeft": 20,
        "marginRight": 20,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Tap heart rate at 
       to see the results here.
    </Text>
  </View>
</View>
`;
